ReSwift-Thunk.xcodeproj/project.pbxproj

```diff
@@ -35,6 +35,8 @@
 		65A3D6EF218B8A300075CB92 /* Thunk.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Thunk.swift; sourceTree = "<group>"; };
 		65A3D6F2218C91F20075CB92 /* ReSwift.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = ReSwift.framework; path = Carthage/Build/iOS/ReSwift.framework; sourceTree = "<group>"; };
 		65A3D6F4218C92420075CB92 /* createThunksMiddleware.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = createThunksMiddleware.swift; sourceTree = "<group>"; };
+		D4286E7721EFED3F00D5749B /* CHANGELOG.md */ = {isa = PBXFileReference; lastKnownFileType = net.daringfireball.markdown; path = CHANGELOG.md; sourceTree = "<group>"; };
+		D4286E7821EFED4D00D5749B /* README.md */ = {isa = PBXFileReference; lastKnownFileType = net.daringfireball.markdown; path = README.md; sourceTree = "<group>"; };
```

Comment:
Do we need to include these `.md` files in the app bundle?

Reply:
@danielmartinprieto They're included in the xcode project simply for ease of access, but unchecked for inclusion in the target, so they do not get included in the framework bundle.

Reply:
Got it 👍